contract/tests/test_contract.py

```diff
@@ -17,7 +17,7 @@ def to_date(date):
     return fields.Date.to_date(date)
 
 
-class TestContractBase(common.SavepointCase):
+class TestContractBase:
```

Comment:
Please don't remove the base inheritance from here.

Reply:
That's done with purpose as tests for multi company import account base ones that inherits from SavePointCase.

Reply:
But this forces all of the inherited tests to multi-inherit from SavepointCase and this one, being very prompt to be forgotten and not having real tests performed. If you keep it, is there any error?

Reply:
Of course, you cannot inherit from same one twice.

Reply:
Well, I don't like to change this base structure, due to the reasons above. Can you please find another solution?